storage/storage-client/src/lib.rs

```diff
@@ -54,6 +55,63 @@ use storage_proto::{
 };
 use tokio::runtime::Runtime;
 
+pub struct StorageServiceClientSimple {
+    network_client: Mutex<NetworkClient>,
+}
+
+impl StorageServiceClientSimple {
+    pub fn new(server_address: &SocketAddr) -> Self {
+        Self {
+            network_client: Mutex::new(NetworkClient::new(*server_address)),
+        }
+    }
+
+    fn request(
+        &self,
+        input: storage_interface::StorageMsg,
```

Comment:
`StorageMsg` should really be named `StorageRequest`, right?

Reply:
Sure.